robotiq_hande_driver/hardware/include/robotiq_hande_driver/hande_hardware_interface.hpp

```diff
@@ -35,14 +37,23 @@ class RobotiqHandeHardwareInterface : public HWI::SystemInterface {
         return *logger_;
     }
 
-   private:
-    // TODO(modbus integration): composition of the modbus communication
+private:
+    GripperApplication application_layer_;
```

Comment:
Let's rename it to something more self-explainable:
```suggestion
    GripperApplication gripper_driver_;
```

Reply:
Done.